sycl/source/detail/pi.cpp

```diff
@@ -74,18 +78,21 @@ void *loadPlugin(const std::string &PluginPath) {
 // needs to setup infrastructure to route PI_CALLs to the appropriate plugins.
 // Currently, we bind to a singe plugin.
 bool bindPlugin(void *Library) {
```

Comment:
Unsure about the calling-sites of this though passing an explicit plugin argument in would be clearer instead of changing a global variable.